CMakeLists.txt

```diff
@@ -31,8 +31,7 @@ project(microxrcedds VERSION "1.2.0")
 
 set(_client_tag v1.1.5)
 set(_agent_tag v1.1.6)
-set(_gen_tag v1.0.2)
-set(_tests_tag v1.1.1)
+set(_gen_tag develop)
```

Comment:
Why do you change this tag? Is it not working?
Please, enable generator tests with `UXRCE_ENABLE_GEN` in CI.

Reply:
WIP the idea is to have all in develop working and test passing.